actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -1012,14 +1012,14 @@ def build_options(selected, options = {})
         end
 
         # Build select option HTML for year.
-        # If <tt>year_format</tt> option is not passed
+        # If <tt>year_format</tt> option is not passed to DateTimeSelector
         #  build_year_options(1998, start: 1998, end: 2000)
         #  => "<option value="1998" selected="selected">1998</option>
         #      <option value="1999">1999</option>
         #      <option value="2000">2000</option>"
         #
-        # If <tt>year_format</tt> option is passed
-        #  build_year_options(1998, start: 1998, end: 2000, year_format: ->year { "Heisei #{ year - 1988 }" })
+        # If <tt>year_format: ->year { "Heisei #{ year - 1988 }</tt> option is passed to DateTimeSelector
```

Comment:
Also remove this change. The example should be enough.

Reply:
I reverted the change.